### PR TITLE
Add ZapDisable description

### PR DESCRIPTION
**Pull request by BruceForstall**
Also, update some of the text, and fix an issue with the useLegacyJit config file syntax.

### CODE REVIEW

**Comment by BruceForstall** [comment]
@richlander @cmckinsey PTAL

File: docs/testing-with-ryujit.md [hunk 1]
```diff
@@ -103,7 +97,52 @@ are specified without using the `.dll` file name extension.
...
   **Note** This is a semicolon-delimited list of simple assembly names.
+
+Disabling the use of all NGEN images
+====================================
+
+To prevent any NGEN native image from being used, and force all code to be compiled with the JIT compiler, you can use the ZapDisable configuration variable, as follows. You might choose to do this as an experiment, to see if any NGEN native image contains generated code that is inducing a bug in your application. Generally, if an NGEN native image does have a problem, and the identity of that native image can be determined, using one of the `DisableNativeImageLoadList` mechanisms described above is preferable.
+
+**Note** This setting applies to both the 32-bit and 64-bit JIT. Thus, setting this globally will affect all 32-bit .NET applications as well. This is particularly true for **Method 2: environment variable**.
+
+**Note 2** NGEN provides significant performance improvements to .NET applications. Disabling the use of NGEN will cause the perfomance of .NET applications to be significantly worse.
```
Comment (cmckinsey) [inline docs/testing-with-ryujit.md]: "Disabling the use of NGEN may result in significantly slower application startup times"

**Comment by cmckinsey** [comment]
Just the one change for the above.

Should we also include a section on "COMPLUS_JitMinOpts" along the methodimpl suggestion for broader opt disabling rather than just the one?

**Comment by BruceForstall** [comment]
@cmckinsey I updated the PR; PTAL

**Comment by cmckinsey** [comment]
Thanks for the updates. LGTM.

**Comment by cmckinsey** [comment]
Should this be in docs section here in dotnet or dotnet\coreclr?

**Comment by BruceForstall** [comment]
I'm just changing the doc the Rich started here. I'm not entirely sure what this repo is intended for. Also, we can't easily change the location (or even name of the doc or name of the sections) because it is the target of a number of external links now (I'm not sure where or how many at this point).

File: docs/testing-with-ryujit.md [hunk 2]
```diff
@@ -136,7 +175,22 @@ You can selectively disable JIT optimization of a particular function by annotat
...
+Disabling optimization of all functions
+=======================================
+
+While troubleshooting, it might be useful to determine if *any* JIT optimization is causing problems. Instead of using the above `MethodImplAttribute` method above, you can set a single environment variable to disable all JIT optimization. Set this variable:
+
+        COMPLUS_JitMinOpts=1
+
+and run your application.
+
+Note that when you run your application under the Visual Studio debugger, it disables JIT optimization by default, to improve the debugging experience. You can read about that [here](https://msdn.microsoft.com/en-us/library/ms241594.aspx).
```
Comment (davkean) [inline docs/testing-with-ryujit.md]: This isn't true anymore, "Suppress JIT optimizations on module load" is off by default in 2015.

Comment (BruceForstall) [inline docs/testing-with-ryujit.md]: Apparently so. The referenced MSDN document is wrong, then.

**Pull request merged**